packages/sitecore-jss-nextjs/src/components/NextImage.test.tsx

```diff
@@ -0,0 +1,262 @@
+/* eslint-disable no-unused-expressions */
+import chai from 'chai';
+import chaiString from 'chai-string';
+import { mount } from 'enzyme';
+import React from 'react';
+import { NextImage, loader } from './NextImage';
+import { ImageField } from '@sitecore-jss/sitecore-jss-react';
+import { ImageLoader, ImageLoaderProps } from 'next/image';
+import { stub } from 'sinon';
+
+const customLoader = () => new URL('https://cm.jss.localhost').href;
+const mockLoader = (stub(customLoader) as unknown) as ImageLoader;
+
+const expect = chai.use(chaiString).expect;
+describe('<NextImage />', () => {
+  describe('Next loader function', () => {
+    it('should append configPath when src is relative', () => {
+      const params: ImageLoaderProps = {
+        config: {
+          deviceSizes: [],
+          imageSizes: [],
+          path: 'https://cm.jss.localhost',
+          allSizes: [],
+          loader: 'default',
+          domains: [],
+          disableStaticImages: false,
+          minimumCacheTTL: 1,
+          formats: [],
+          dangerouslyAllowSVG: false,
+          contentSecurityPolicy: 'test',
+        },
+        src: '/assets/img/test0.png',
+        width: 100,
+      };
+
+      const result = loader(params);
+      expect(result).to.be.a('string');
+      expect(result).to.equal(`${params.config.path}/assets/img/test0.png?mw=100`);
+    });
+
+    it('should not require config path when src is absolute', () => {
```

Comment:
This test still doesn't look quite right. Let's create 2 tests for absolute:

1. 'should not require config path when src is absolute' -> with the config.path = undefined
2. 'should not append config path when src is absolute' -> with the config.path = 'https://cm.jss.localhost/'